From 50f3fec3377c26061e3c309a66ea9906ad375f67 Mon Sep 17 00:00:00 2001
From: Edgar E. Iglesias <edgar.iglesias@gmail.com>
Date: Sun, 21 Aug 2011 22:23:04 +0200
Subject: [PATCH 04/79] testsuite: Define MicroBlaze PIC register

2011-08-21  Edgar E. Iglesias  <edgar.iglesias@gmail.com>

	* gcc.dg/20020312-2.c: Add definition for MicroBlaze.

Signed-off-by: Edgar E. Iglesias <edgar.iglesias@gmail.com>
---
 gcc/testsuite/gcc.dg/20020312-2.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/gcc/testsuite/gcc.dg/20020312-2.c b/gcc/testsuite/gcc.dg/20020312-2.c
index 0689636..a37cef2 100644
--- a/gcc/testsuite/gcc.dg/20020312-2.c
+++ b/gcc/testsuite/gcc.dg/20020312-2.c
@@ -40,6 +40,8 @@ extern void abort (void);
 /* No pic register.  */
 #elif defined(__mcore__)
 /* No pic register.  */
+#elif defined(__MICROBLAZE__)
+# define PIC_REG "r20"
 #elif defined(__mips__)
 /* PIC register is $28, but is used even without -fpic.  */
 #elif defined(__MMIX__)
-- 
1.7.1

